src/patternfly/components/TreeView/tree-view-link-icon.hbs

```diff
@@ -0,0 +1,14 @@
+<span class="pf-c-tree-view__link-icon{{#if tree-view-link-icon--modifier}} {{tree-view-link-icon--modifier}}{{/if}}"
+  {{#if tree-view-link-icon--attribute}}
+    {{{tree-view-link-icon--attribute}}}
+  {{/if}}>
+  {{#if tree-view-link-icon--IsFolder}}
+    {{#if tree-view-link-icon--IsExpanded}}
```

Comment:
I think you could write this in a single line via

`<i class="fas fa-folder{{#if tree-view-link-icon--IsExpanded}}-open{{/if}}" aria-hidden="true"></i>